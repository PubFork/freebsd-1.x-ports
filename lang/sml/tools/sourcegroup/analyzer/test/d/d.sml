local open A in structure T = B end
