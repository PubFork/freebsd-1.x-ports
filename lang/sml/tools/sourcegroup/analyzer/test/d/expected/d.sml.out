source sml "d/d.sml"
  import structure A B 
  export structure T ;

